README.md

```diff
@@ -30,11 +30,13 @@ The elements shown in the architecture diagram above are deployed by the [Cyral
 ### Requirements
 
 * Make sure you have access to your AWS environment with an account that has sufficient permissions to deploy the sidecar. The minimum permissions must allow for the creation of the elements listed previously. We recommend Administrator permissions (`AdministratorAccess` policy) as the module creates an IAM role.
+* Decide your certificate deployment type. Refer to the [Sidecar certificates](./docs/certificates.md) document for additional guidance.
 
 ### Examples
 
 #### Quick Start
 
+* (Optional) Create/Import your own certificate(s) to be used by this template.
```

Comment:
This should be in the instructions for a `production` sidecar.

Reply:
still the same as above